packages/KButton/KButton.spec.js

```diff
@@ -80,4 +108,31 @@ describe('KButton', () => {
     expect(wrapper.find('a').attributes('href')).toBe('https://google.com')
     expect(wrapper.html()).toMatchSnapshot()
   })
+
+  it('renders an icon when using icon prop', () => {
+    const wrapper = mount(KButton, {
+      propsData: {
+        icon: 'spinner'
+      },
+      slots: {
+        default: `Click Me`
+      }
+    })
+
+    expect(wrapper.find('.k-button .k-button-icon.kong-icon-spinner').exists()).toBeTruthy()
```

Comment:
```suggestion
    expect(wrapper.find('.k-button .kong-icon-spinner').exists()).toBeTruthy()
```
I think the class is nested so just check for this